### ISSUE TITLE
Output dimension

### ISSUE THREAD

**Issue by AziziShekoofeh** [jakeret/tf_unet#183]
Hi,

I am trying to use your package for training a unet model on a dataset includes jpeg images with the dimensions of 128x128x3 and the binary mask of 128x128x2. The output which I can read from the network as the logits  (node "div_1") has the dimension of 36x36x2. These logits then after being used as the input to the pixel_wise_softmax_2 layer to generate the "predictor" and finally to calculate the error rate. To align the size of the ground-truth and the prediction you also used crop_to_shape which is also a bit weird to me.   As I saw in many other implementations unet supposed to generate the same size result. Any insight to this problem or how to solve it? I expect to see an upsampling which I couldn't find it.  

Best,
Shekoofeh
